### ISSUE TITLE
input keyboard opens and suddenly closes dramatically 

### ISSUE THREAD

**Issue by mohammedelnady** [oneHamidreza/MeowBottomNavigation#52]

**Comment by mohammedelnady** [comment]
please help

**Comment by mohammedelnady** [comment]
there is no reply for help Are You ALIVE ?!

**Comment by mohammedelnady** [comment]
are you alright ?!

**Comment by oneHamidreza** [comment]
please send more info from issue such as screen-shot or your codes

**Comment by mohammedelnady** [comment]
https://drive.google.com/file/d/1PZksbrpI6ttHIjXiiLc9du9xemAcPTwY/view?usp=drivesdku

I uploaded aPreview inwhich when I opens keyboard ,it closed permanently everytime

**Comment by oneHamidreza** [comment]
ok . i will check it.

**Comment by oneHamidreza** [comment]
Please send your codes. Are you sure to this issue is related with Meow Bottom Navigation?

**Comment by mohammedelnady** [comment]
الصبر يارب
Yes it's abug in your library 
And you assigned here as abug
https://github.com/oneHamidreza/MeowBottomNavigation/issues/45

Are you ok ??

**Comment by DevUai** [comment]
Any solution? I'm having the same problem with the Java version.

**Comment by newarifrh** [comment]
try to add this code in your manifest like this way
```
 <activity android:name=".MainActivity"
        android:windowSoftInputMode="adjustPan">
```

**Comment by DevUai** [comment]
Unfortunately it didn't work. It keeps opening and closing instantly.

**Comment by mohammedelnady** [comment]
any recommendations ?

**Comment by mohammedelnady** [comment]
any updates?!

**Comment by mohammedelnady** [comment]
hey?!

**Comment by mohammedelnady** [comment]
Are You Alive ?

**Comment by mohammedelnady** [comment]
Hey

**Comment by ashokgaire** [comment]
same problem here

**Comment by abhishektiwari-kiwi** [comment]
Just take all code from onMeasure() method in MeowBottomNavigation class i.e. and paste it at the bottom of method initializeViews() after line : allowDraw = true.
And Voila!

**Comment by abhishektiwarijr** [comment]
Just take all code from onMeasure() method in MeowBottomNavigation class and paste it in the method initializeViews() at the bottom after line allowDraw = true.
        And Voila!

**Comment by Shamseldienmahmoud** [comment]
> Just take all code from onMeasure() method in MeowBottomNavigation class and paste it in the method initializeViews() at the bottom after line allowDraw = true.
> And Voila!

not working bro

**Comment by Shamseldienmahmoud** [comment]
Any solution? same problem with the Java version.

**Comment by abhishektiwarijr** [comment]
Well, I've tried kotlin version and I don't think it should matter with the language.
You can see below.

![ezgif com-optimize](https://user-images.githubusercontent.com/25899457/80785633-8fb02200-8b9e-11ea-88ea-7044a6ae1a21.gif)

**Comment by mahdijahandide** [comment]
add windowSoftInput="adjust_resize" in manifest and it will be fixed

**Comment by Shamseldienmahmoud** [comment]
> add windowSoftInput="adjust_resize" in manifest and it will be fixed

not working

**Comment by abhishektiwarijr** [comment]
@Shamseldienmahmoud 
This didn't work https://github.com/oneHamidreza/MeowBottomNavigation/issues/52#issuecomment-622261915 ?
And I hope your activity has this snippet  
android:configChanges="keyboardHidden|orientation|screenSize" 
in manifest
Let us know if there's anything else.

**Comment by oneHamidreza** [comment]
The Sample in Project doesn't have issue with an EditText.

**Comment by oneHamidreza** [comment]
Check `v1.3.1`.

**Comment by komal80501** [comment]
> https://drive.google.com/file/d/1PZksbrpI6ttHIjXiiLc9du9xemAcPTwY/view?usp=drivesdku
> 
> I uploaded a Preview inwhich when I open keyboard ,it closes permanently everytime

I was also facing some issue, 
I have solved this by changing just 


> https://drive.google.com/file/d/1PZksbrpI6ttHIjXiiLc9du9xemAcPTwY/view?usp=drivesdku
> 
> I uploaded a Preview inwhich when I open keyboard ,it closes permanently everytime

I was also facing some issue
Just check which change listener

I have used setOnShowListner 
bottomNavigation.setOnShowListener {
    // YOUR CODES
}

bottomNavigation.setOnClickMenuListener {
    // YOUR CODES
}




> https://drive.google.com/file/d/1PZksbrpI6ttHIjXiiLc9du9xemAcPTwY/view?usp=drivesdku
> 
> I uploaded a Preview inwhich when I open keyboard ,it closes permanently everytime

Have faced Same issue
I have resolved by changing the listener
bottomNavigation.setOnShowListener {
    // YOUR CODES
}
Use setOnClickMenuListener() function to access when a cell has been clicked.

bottomNavigation.setOnClickMenuListener {
    // YOUR CODES
}

I have Used above listener and it will work for me....hope this will work for you as well

**Comment by fkocak2505** [comment]
> https://drive.google.com/file/d/1PZksbrpI6ttHIjXiiLc9du9xemAcPTwY/view?usp=drivesdku
> 
> > I uploaded a Preview inwhich when I open keyboard ,it closes permanently everytime
> 
> Have faced Same issue
> 
> bottomNavigation.setOnClickMenuListener {
> // YOUR CODES
> }
> 
> I have Used above listener and it will work for me....hope this will work for you as well

yeah it's working. I have same issue but when I use "setOnClickMenuListener" than fixing. Thanks..!

**Comment by MegiasHack** [comment]
Solution for java?

**Comment by MegiasHack** [comment]
> Solution for java?

If anyone had the problem on java, you need to use click listener instead show listener and leave show listener empty and it will work.

**Comment by komalM01** [comment]
bottomNavigation.setOnClickMenuListener implement this listener

**Comment by Zeyrox1090** [comment]
😔
